### PR TITLE
[iOS #430] Create StaffView

### LINKED ISSUE #430: [iOS] Create StaffView
**Idea Description**

- Create StaffFeature target
- Create StaffView in StaffFeature target
  - UI only is okey 👍🏼 
  - implement StaffView UI with ContributorView design once.

**Reference images and links**
スクリーンショット 2022-09-14 1 59 48

### PR DESCRIPTION
**Pull request by coffmark**
## Issue
- close #430 

## Overview (Required)
- Create StaffFeature target
- Create StaffView
- Fix AboutView to be able to see staff view
- only UI ( fetched data from fakes function )

## Links
- [Figma](https://www.figma.com/file/NcSMs6dMsD88d4wOY0g3rK/DroidKaigi-2022-Conference-App?node-id=372%3A2616)

## Screenshot
https://user-images.githubusercontent.com/52638834/190849190-152e6729-bfb8-469b-984c-778f673c9d18.mov

### CODE REVIEW

File: app-ios/Sources/AboutFeature/AboutView.swift [hunk 1]
```diff
@@ -1,26 +1,76 @@
...
+    case license
+}
+
 public struct AboutState: Equatable {
-    public init() {}
+    public var staffState: StaffState
+    public var isSheetPresented: Bool
+    public var destination: AboutDestination?
+    public init(
+        staffState: StaffState = .init(staffs: Staff.companion.fakes()),
```
[the remark this answers is not in the document]
Comment (coffmark) [reply app-ios/Sources/AboutFeature/AboutView.swift]: Fixed! https://github.com/DroidKaigi/conference-app-2022/pull/543/commits/1e663f1b7ec31ff374372b3a4c4a6dbcfec14bf8 Thanks!

**Review by ry-itto — APPROVED** [review]
LGTM! Thank you!!

**Review by ry-itto — COMMENTED** [review]
CI failed with `StaffFeature` target doesn't depend on `AboutFeature`.
So please add `StaffFeature` to `AboutFeature`'s dependencies 🙏🏼

**Comment by coffmark** [comment]
Hello, @ry-itto !
Sorry for late reply.
I'm having a conflict, do you want it to be displayed in Modal or in NavigationView ?

**Review by ry-itto — APPROVED** [review]
LGTM! Thanks!

**.xcscheme files may be removed later by adding .gitignore. But it's okey it's not in .gitignore for now 🙏🏼

**Pull request merged**